guidance/credential-sharing.md

```diff
@@ -1,27 +1,21 @@
+Whenever possible, no account should be shared with other users.  However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users by using a tool called **[1Password](https://1password.com)**.
 
- ⚠️ Credentials should **never** be shared outside LastPass: please **don't** send passwords in plaintext by any other means.
+The DevOps and Infosec teams maintain a master account where all credentials are stored and shared with the approved users. Passwords are being rotated regularly for security reasons, and the new ones will automatically synchronize in the other users' 1Password vaults. 
 
-## LastPass master account
-The DevOps and Infosec teams maintain a master account where all credentials are stored and shared with the approved users. Passwords are being rotated regularly for security reasons, and the new ones will automatically synchronize in the other users' LastPass vaults.  
+:warning: Credentials should **never** be shared outside the password manager: please **do not** send passwords in plaintext by any other means.
 
-## Creating a LastPass account
-If you do not have a lastpass account please [create one](https://lastpass.com/create-account.php).  **DO NOT FORGET YOUR LASTPASS PASSWORD!**  It cannot be reset by the infosec team or lastpass.  Your password is used to generate your encryption key, so it is unique to you.  We also recommend installing the lastpass browser extension to your favorite browser.
+## :book: Getting started
 
-After your account is created make sure to sign in, this will generate your encryption keys so passwords can be shared securely.  Send the email address you used to sign up to the infosec team ([#skill-infosec](https://app.slack.com/client/TUTSYURT3/CV4TYGC1Z)) and they will share passwords with you.  If you are outside of the admin group in slack approval for access will be required.  The infosec team will reach out to the admin group for approval. 
+Just send your email address to the _Infosec_ team ([#skill-infosec](https://app.slack.com/client/TUTSYURT3/CV4TYGC1Z)) and you'll be added to 1Password with access to the shared secrets you're allowed to use. If you already have a 1Password account, use its email address when requesting access.
 
-[Lastpass sharing documentation](https://helpdesk.lastpass.com/sharing-4-0/)
+If you aren't part of the [@admin-group](https://helpfulengineering.slack.com/admin/user_groups) in Slack, approval for access will be required.
 
-## Changing passwords
-When changing a password, you should [generate the new one with LastPass](https://helpdesk.lastpass.com/generating-a-password/) (or equivalent) using the following settings:
+Now, you can proceed to set up your browser and devices following this guide: https://support.1password.com/explore/team-member/
 
-* **All characters**:
```

Comment:
Deleted the password generation settings because the new password manager has pretty sensible defaults